### DISCUSSION TITLE
Hide chords from the player?

### DISCUSSION THREAD

**Discussion by AdamSEY** [CoderLine/alphaTab#1205] [Q&A] [answered]
Is there's any way we could choose whether we want to show or hide chords? Also maybe we don't show duplicates?

<img width="892" alt="Screenshot 2023-06-29 at 22 28 58" src="https://github.com/CoderLine/alphaTab/assets/28712999/20917c0f-430a-4f61-aaae-bd8dcc72f087">

**Comment by Danielku15** [comment, marked as answer]
> Is there's any way we could choose whether we want to show or hide chords? 
Chord diagrams can be hidden via https://alphatab.net/docs/reference/settings/notation/elements#description

> Also maybe we don't show duplicates?

Do you have an example file for your screenshot? Normally the input file (guitar pro) should already handle that chords are generally unique. Maybe there is something special to the chords which is not visible in alphaTab but makes them different. Are you maybe rendering multiple tracks at the same time?

**Reply by AdamSEY** [reply]
Thank you for your prompt response, it's truly appreciated. I have attached the file and it seems like it's happening quite often.

[Live_and_Let_Die_5RldfoFmPL.gp5.zip](https://github.com/CoderLine/alphaTab/files/11929037/Live_and_Let_Die_5RldfoFmPL.gp5.zip)

**Reply by Danielku15** [reply]
I added https://github.com/CoderLine/alphaTab/issues/1209 to track this improvement.
